source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -190,16 +190,10 @@ int DynamicMetadataMapWrapper::luaPairs(lua_State* state) {
 }
 
 int PublicKeyWrapper::luaGet(lua_State* state) {
-  if (public_key_ == nullptr || public_key_.get() == nullptr) {
+  if (public_key_.empty()) {
     lua_pushnil(state);
   } else {
-    auto wrapper = Common::Crypto::Access::getTyped<Common::Crypto::PublicKeyObject>(*public_key_);
-    EVP_PKEY* pkey = wrapper->getEVP_PKEY();
-    if (pkey == nullptr) {
-      lua_pushnil(state);
-    } else {
-      lua_pushlightuserdata(state, public_key_.get());
```

Comment:
WDYT about having a format checker which blocks this API? I think we should never use it or at least make someone turn off the linter?